engine-src/Game/LambdaHack/Client/UI/Watch/WatchSfxAtomicM.hs

```diff
@@ -155,7 +156,7 @@ watchRespSfxAtomicUI sfx = case sfx of
           -- out only in Allure.
           when (isOurCharacter
                 && Ability.getSk Ability.SkArmorMelee actorMaxSk > 0) $
-            msgAdd MsgTutorialHint "You took damage of a different kind than the normal piercing hit, which means your armor couldn't block any part of it. Normally, your HP (hit points, health) do not regenerate, so losing them is a big deal. Apply healing concoctions or take a long sleep to replenish your HP (but in this hectic environment not even uninterrupted resting that leads to sleep is easy)."
+            msgAdd MsgTutorialHint  (tshow DamageOfDifferentKind)
```

Comment:
In the long run, probably best to define a new funcion that given only one argument `DamageOfDifferentKind` does all of the above.